docs/pages/api-docs/data-grid/data-grid-pro.md

```diff
@@ -228,7 +228,7 @@ You can override the style of the component thanks to one of these customization
 - With a [global class name](/customization/components/#overriding-styles-with-global-class-names).
 - With a theme and an [`overrides` property](/customization/globals/#css).
 
-If that's not sufficient, you can check the [implementation of the component style](https://github.com/mui-org/material-ui-x/blob/master/packages/grid/_modules_/grid/components/containers/GridRootStyles.ts) for more detail.
+If that's not sufficient, you can check the [implementation of the component style](https://github.com/mui-org/MUI-x/blob/master/packages/grid/_modules_/grid/components/containers/GridRootStyles.ts) for more detail.
```

Comment:
404

```suggestion
If that's not sufficient, you can check the [implementation of the component style](https://github.com/mui-org/material-ui-x/blob/next/packages/grid/_modules_/grid/components/containers/GridRootStyles.ts) for more detail.
```

Could you also point to next below?

https://github.com/mui-org/material-ui-x/blob/71cc0d3e6471cd6068a8fba555620953c553a377/docs/pages/api-docs/data-grid/data-grid.md#L219